versioned_docs/version-2.8/integrations-in-rancher/cloud-marketplace/aws-cloud-marketplace/install-adapter.md

```diff
@@ -19,6 +19,7 @@ In order to deploy and run the adapter successfully, you need to ensure its vers
 
 | Rancher Version | Adapter Version  |
 |-----------------|:----------------:|
+| v2.9.0          | v104.0.0+up4.0.0 |
```

Comment:
```suggestion
```

Since this is for v2.9.0, it should be made to versioned_docs/version-2.9/integrations-in-rancher/cloud-marketplace/aws-cloud-marketplace/install-adapter.md instead. The file being modified here is intended for v2.8 listings.

Reply:
done